core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -153,6 +155,7 @@
 
    @GuardedBy("lock#readLock")
    private <K, V> NonBlockingStore<K, V> getStoreLocked(Predicate<StoreStatus> predicate) {
+      if (stores == null) return null;
```

Comment:
Nitpick: I don't think we put the return on the same line as the if elsewhere, and we usually leave an empty line after it.

```suggestion
      if (stores == null)
         return null;

```

Reply:
Sure

Reply:
I didn't mean to add braces as well, but I'll take it :)